internal/provider/inventory_resource.go

```diff
@@ -95,17 +96,15 @@ func (r *InventoryResource) Schema(_ context.Context, _ resource.SchemaRequest,
 			"organization": schema.Int64Attribute{
 				Computed: true,
 				Optional: true,
+				Default:  int64default.StaticInt64(1),
 				PlanModifiers: []planmodifier.Int64{
 					int64planmodifier.UseStateForUnknown(),
 				},
 				Description: "Identifier for the organization the inventory should be created in. " +
 					"If not provided, the inventory will be created in the default organization.",
 			},
 			"organization_name": schema.StringAttribute{
-				Computed: true,
-				PlanModifiers: []planmodifier.String{
-					stringplanmodifier.UseStateForUnknown(),
-				},
+				Computed:    true,
```

Comment:
What's the reason for this change? I think we have some newly-reported issues reported around PlanModifiers and validation around Unknown values after recent changes, so it would be good to know why we're changing the PlanModifiers here. Was there some issue with the `organization_name` that we're addressing?

Reply:
If I have the `useStateForUnknown` plan modifier here I get this error:

```
 inventory_resource_test.go:178: Step 3/4 error: Error running apply: exit status 1
        
        Error: Provider produced inconsistent result after apply
        
        When applying changes to aap_inventory.test, provider
        "provider[\"registry.terraform.io/hashicorp/aap\"]" produced an unexpected
        new value: .organization_name: was cty.StringVal("Default"), but now
        cty.StringVal("Default2").
```

The issue is TF uses the previous state for this value because the configuration doesn't defined the organization_name.

Reply:
Thanks for checking. That makes sense.